chinese_tutorial_rvest_httr.Rmd

```diff
@@ -0,0 +1,490 @@
+
+#Chinese Web scraping using R package: rvest & httr tutorial
+
+Huiqian Yu
+
+```{r install, eval=FALSE, include=FALSE}
```

Comment:
The file should not contain any install.packages(), write functions, setwd(), or getwd()